### ISSUE TITLE
Add Entry Point for Status Page on Authenticated "Get Reimbursed for Travel Pay" page

### ISSUE THREAD

**Issue by micahtaylor** [department-of-veterans-affairs/va.gov-team#83284]
## Description
Add an entry point to https://staging.va.gov/health-care/get-reimbursed-for-travel-pay/

## Acceptance Criteria

- [ ] Engineering Implementation
     - [ ] Unauth page
     - [ ] Auth page
     - [ ] Contains MHV copy until 1/31 

## Definition of Done
- [ ] All acceptance criteria are met
- [ ] Documentation is updated

**Comment by ayush-chak** [comment]
Slack thread here: https://dsva.slack.com/archives/C05UTPZRZFY/p1718208029294709

**Comment by carafrissell1** [comment]
Other entry point to enable: Claims tool from My VA 

<img width="1750" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMTViQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--bf9fb252bdbfc9370ff721a9436b8e0bd83a70dd/Screenshot%202024-06-12%20at%201.21.57%E2%80%AFPM.png" alt="Screenshot 2024-06-12 at 1.21.57 PM.png" />

**Comment by ayush-chak** [comment]
Based on conversations with Kristen and Kay, we are moving this to low-priority - we can revisit additional entry points after rollout.

**Comment by sterkenburgsara** [comment]
Question - I talked Cara + Micah yesterday in the appointments & travel pay ux sync about adding an entry point to the unauth benefit hub page for travel pay. But we didn't discuss adding one to the My VA page (which this ticket seems to indicate). Could you clarify? 

Adding another link to the My VA page will be a bigger can of worms, b/c auth experience governance will limit the number of links we can have under the "Health care" section, and we already have 5 listed there (My HealtheVet landing page + 4 "big" health tools). Some changes are currently pending to this section now (updating links on My VA to point to the new My HealtheVet on VA.gov) Happy to discuss that in more depth at any point if helpful.

**Comment by ayush-chak** [comment]
Hi @sterkenburgsara - my apologies, I think I used the wrong terminology, but I just updated the ticket name and included the URL directly - even though I accessed it through MyVA, I believe it's a different page, so that's my mistake.

Can you confirm that we would be able to add the entry point to https://staging.va.gov/health-care/get-reimbursed-for-travel-pay/ in both the auth and unauth states? Thank you!

**Comment by sterkenburgsara** [comment]
@ayush-chak No worries, I just wanted to make sure we were talking about the same thing & meeting your team's needs. 

We can definitely add an entry point to the tool on that page. However, because your tool requires: 1.) a facility in the profile and 2.) LOA3 status in order to display and render data to the user, your application has to sit _behind_ authentication. We should not display this link to unauthenticated users for whom it will be a dead-end. 

Instead, unauth users will need to see a variety of sign-in alert widgets on the unauth page. Those alerts [are in VADS here](https://design.va.gov/components/alert/alert-sign-in/) & are a new standard across VA.gov. The caveat (more for @kjduensing) is that they exist in Figma, but aren't quite yet built out yet. We're waiting on the identity team to create these forced id-verification buttons for Login.gov and ID.me flows & don't have a final delivery date for that work at this time. They are aware of the urgency & many teams are dependent on the work. I can circle back with you once we get timelines firmed up, though!

**Comment by ayush-chak** [comment]
Excellent, thanks so much for the info. So just so we are clear for planning purposes:

* the auth work is ready for us to work on, so we can start that
* unauth will need some time before ready to implement

Is that correct?

**Comment by carafrissell1** [comment]
Hi I think there could be some confusion.  @laurwill and I have been talking about this sync in [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/96858#issuecomment-2486566895).

**Comment by carafrissell1** [comment]
@laurwill can you add your comments here?  adding the relevant thread from the other ticket

Sorry for the 2 screenshots - it's a long page, but this is what a user sees when they are [logged on this page.](https://staging.va.gov/health-care/get-reimbursed-for-travel-pay/)
![Image](https://github.com/user-attachments/assets/f873bb6a-d9f2-46dd-a175-3144e5208136)
![Image](https://github.com/user-attachments/assets/8b1f0241-fb25-4a67-831e-9f456929909c)

**Comment by sterkenburgsara** [comment]
@ayush-chak yes, there is a blocker in the way of the unauth engineering work at the moment.

**Comment by ayush-chak** [comment]
Shuffling - this will now be a ticket for engineering discussion and implementation _only._ 

Sara, I will tag you in the relevant ticket for further discussion to avoid any confusion. Thank you so much for your help so far already!
